components/aws/sagemaker/tests/integration_tests/utils/minio_utils.py

```diff
@@ -0,0 +1,39 @@
+import utils
+import os
+
+from minio import Minio
+
+
+def get_artifact_in_minio(workflow_json, step_name, artifact_name, output_dir):
```

Comment:
Add docstrings

Reply:
I am working on another PR to add more tests, will address these then